src/applications/vaos/appointment-list/components/RequestedAppointmentsList.jsx

```diff
@@ -1,28 +1,21 @@
 import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { shallowEqual, useDispatch, useSelector } from 'react-redux';
-
 import recordEvent from 'platform/monitoring/record-event';
```

Comment:
Fix eslint warning.
import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports';